tests/test_visitors/test_ast/test_modules/test_magic_module_functions.py

```diff
@@ -10,21 +10,32 @@
     MagicModuleFunctionsVisitor,
 )
 
-regular_function = """
+module_level_method = """
 def {0}(name):
     ...
 """
 
+class_level_method = """
+class Example(object):
+    def {0}(name):
+        ...
+"""
+
 
+@pytest.mark.parametrize('code', [
+    module_level_method,
+    class_level_method,
+])
 @pytest.mark.parametrize('function_names', MAGIC_MODULE_NAMES_BLACKLIST)
-def test_wrong_magic_used(
+def test_right_magic_used(
```

Comment:
I found another typo here (it should be test_right_magic_used since it tests correct cases)